src/Plugin/LessPlugin.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * @copyright 2017 Hostnet B.V.
+ */
+declare(strict_types=1);
+namespace Hostnet\Component\Resolver\Plugin;
+
+use Hostnet\Component\Resolver\Bundler\Processor\LessContentProcessor;
+use Hostnet\Component\Resolver\Bundler\Runner\LessRunner;
+use Hostnet\Component\Resolver\Cache\CachedImportCollector;
+use Hostnet\Component\Resolver\ConfigInterface;
+use Hostnet\Component\Resolver\Import\BuiltIn\LessImportCollector;
+use Hostnet\Component\Resolver\Import\Nodejs\Executable;
+use Psr\SimpleCache\CacheInterface;
+
+class LessPlugin implements PluginInterface
```

Comment:
There should be some docs explaining which less we mean, what problem this solves, which node package that is, maybe a link to the website of less, ...

Ditto for other plugins